index.html

```diff
@@ -0,0 +1,666 @@
+                        </p>
+                    </div>
+                    <img src="img/image2.jpeg" alt="article picture">
+                </div>
+                <ul class="thin">
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                </ul>
+            </div>
+            <div class="more-articles">
+                <p class="orange-headers">Hard Fork <i class="fas fa-chevron-right"></i></p>
+                <img class="header-img" src="img/latest-deals.png" alt="latest deals">
+                <div class="more-info-header">
+                    <div>
+                        <p class="orange thin">
+                            LATEST
+                        </p>
+                        <p class="orange">
+                            How your company can limit its liability for US data collection lawsuits
+                        </p>
+                    </div>
+                    <img src="img/image2.jpeg" alt="article picture">
+                </div>
+                <ul class="thin">
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                    <li>Has Jeff Bezos ended world hunger? Twitter account skewers Amazon chief's absurd $178B fortune</li>
+                </ul>
+            </div>
+
+        </section>
+
+        <div class="news-pages">
+        <h2 class="orange news-pages-header">Latest News <i class="fas fa-chevron-right"></i></h2>
+        <section class="latest-news">
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
```

Comment:
- [ ] It should be Latest Deals

- [ ] The alignment and image sizes seem off. You also missed the 'see all deals' on the top right.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88701766-da2f3500-d0e0-11ea-82c3-4ce2c3b5536f.png) | ![image](https://user-images.githubusercontent.com/61479648/88701791-e3b89d00-d0e0-11ea-9a36-5fdfe1aaf3e2.png)|